core-it-support/core-it-javaagent/pom.xml

```diff
@@ -53,6 +53,17 @@ under the License.
             <goals>
               <goal>shade</goal>
             </goals>
+            <configuration>
+              <filters>
+                <filter>
+                  <artifact>*:*</artifact>
+                  <excludes>
+                    <exclude>module-info.class</exclude>
+                    <exclude>META-INF/MANIFEST.MF</exclude>
+                  </excludes>
+                </filter>
+              </filters>
+            </configuration>
```

Comment:
"Fixes" these warnings:
```
[INFO] --- maven-shade-plugin:3.4.1:shade (default) @ core-it-javaagent ---
[INFO] Including org.ow2.asm:asm:jar:7.3.1 in the shaded jar.
[INFO] Including org.ow2.asm:asm-commons:jar:7.3.1 in the shaded jar.
[INFO] Including org.ow2.asm:asm-tree:jar:7.3.1 in the shaded jar.
[INFO] Including org.ow2.asm:asm-analysis:jar:7.3.1 in the shaded jar.
[INFO] Including org.ow2.asm:asm-util:jar:7.3.1 in the shaded jar.
[INFO] Dependency-reduced POM written at: /home/psiroky/code/maven-ecosystem/maven-integration-testing/core-it-support/core-it-javaagent/dependency-reduced-pom.xml
[WARNING] Discovered module-info.class. Shading will break its strong encapsulation.
[WARNING] Discovered module-info.class. Shading will break its strong encapsulation.
[WARNING] Discovered module-info.class. Shading will break its strong encapsulation.
[WARNING] Discovered module-info.class. Shading will break its strong encapsulation.
[WARNING] Discovered module-info.class. Shading will break its strong encapsulation.
[WARNING] asm-7.3.1.jar, asm-analysis-7.3.1.jar, asm-commons-7.3.1.jar, asm-tree-7.3.1.jar, asm-util-7.3.1.jar, core-it-javaagent-2.1-SNAPSHOT.jar define 1 overlapping resource: 
[WARNING]   - META-INF/MANIFEST.MF
[WARNING] maven-shade-plugin has detected that some class files are
[WARNING] present in two or more JARs. When this happens, only one
[WARNING] single version of the class is copied to the uber jar.
[WARNING] Usually this is not harmful and you can skip these warnings,
[WARNING] otherwise try to manually exclude artifacts based on
[WARNING] mvn dependency:tree -Ddetail=true and the above output.
[WARNING] See https://maven.apache.org/plugins/maven-shade-plugin/
```